odh-common/base/odhapplicationcrd.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: odhapplications.applications.odh.io
```

Comment:
Should we specify that this is for the `dashboard` or is there some future plan to implement a generic `odhapplication` that provides some additional functionality beyond the dashboard.

Some variation of `ODHDashboardApplication.dashboard.opendatahub.io`

Reply:
looks good thanks